.changeset/fluffy-cobras-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-css': major
```

Comment:
```suggestion
'@itwin/itwinui-css': patch
```
Following [semantic versioning](https://semver.org/), I believe this will be a patch.